JS/index.js

```diff
@@ -1,37 +1,29 @@
-const Plane = require('./Planes/Plane');
-const MilitaryPlane = require('./Planes/MilitaryPlane');
-const PassengerPlane = require('./Planes/PassengerPlane');
-const Airport = require('./Airport');
-const MilitaryType = require('./models/MilitaryType');
-const experimentalPlane = require('./Planes/experimentalPlane');
-const ExperimentalTypes = require('./models/ExperimentalTypes');
-const ClassificationLevel = require('./models/ClassificationLevel');
+const MilitaryPlane = require('./planes/military-plane');
+const PassengerPlane = require('./planes/passenger-plane');
+const Airport = require('./airport/airport');
+const MILITARY_TYPE = require('./models/military-type');
+const ExperimentalPlane = require('./planes/experimental-plane');
+const EXPERIMENTAL_TYPES = require('./models/experimental-type');
+const CLASSIFICATION_LEVEL = require('./models/classification-level');
 
-(function run() {
+const planesTestGrid = [
```

Comment:
Why did you delete module syntax for this file and just live constant?
You changed the code logic. Please return the initial one.